app/policies/assignment_journal_policy.rb

```diff
@@ -0,0 +1,16 @@
+class AssignmentJournalPolicy < ApplicationPolicy
+  class Scope < ApplicationScope
+    def resolve
+      all if superadmin?
+      resolve_author if volunteer?
```

Comment:
The first line of this method is ignored, add `return` or use a multi-line `if` statement.